File: sdk/go/entity/recon_exception.go
package entity

import (
	"time"
)

// ReconException 对账异常记录
type ReconException struct {
	ID            int64     `json:"id"`
	OrderNo       string    `json:"orderNo"`
	MerchantId    string    `json:"merchantId"`
	ExceptionType int       `json:"exceptionType"`
	ExceptionMsg  string    `json:"exceptionMsg"`
	ExceptionStep int       `json:"exceptionStep"`
	CreateTime    time.Time `json:"createTime"`
	UpdateTime    time.Time `json:"updateTime"`
}

File: sdk/python/entity/recon_exception.py
from dataclasses import dataclass
from datetime import datetime
from typing import Optional

@dataclass
class ReconException:
    """对账异常记录"""
    order_no: str
    merchant_id: Optional[str] = None
    exception_msg: Optional[str] = None
    exception_step: Optional[int] = None
    
    create_time: Optional[datetime] = None
    update_time: Optional[datetime] = None
